### PR TITLE
Feature/style add issue page#28

### PR DESCRIPTION
**Pull request by tobybessant**

### CODE REVIEW

File: src/app/views/landlord/new-issue/new-issue.page.html [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+      <ion-label class="label priority-spinner" color="primary" (click)="showPicker()">{{ priorityString }}</ion-label>
+    </ion-item>
+
+    <ion-item>
+      <ion-label class="label" position="floating">Title</ion-label>
+      <ion-input type="title" [(ngModel)]="title" (ionChange)="checkChange()"></ion-input>
+    </ion-item>
+
+    <ion-item>
+
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.html]: Extra whitespace

File: src/app/views/landlord/new-issue/new-issue.page.scss [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+ion-item {
+  --padding-left: 0;
+}
+
+.toolbar-text {
+  font-size: 13pt;
+}
+
+.priority-input {
+  padding:5px 0 0px 0;
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.scss]: Space needed

File: src/app/views/landlord/new-issue/new-issue.page.scss [hunk 3]
```diff
@@ -1,11 +1,29 @@
 ion-title {
   text-align: center;
+  padding: 0 0 3px 0;
+  margin:0;
```
Comment (tobysmith568) [inline src/app/views/landlord/new-issue/new-issue.page.scss]: Space needed

**Review by tobysmith568 — CHANGES_REQUESTED** [review]
Added a few comments

**Review by tobysmith568 — APPROVED** [review]

**Review by Gubbsy — APPROVED** [review]
Semi-good job frend! 🥇

**Pull request merged**